package.json

```diff
@@ -62,15 +62,17 @@
     "normalize-selector": "^0.2.0",
     "pify": "^3.0.0",
     "postcss": "^6.0.16",
-    "postcss-html": "^0.18.0",
+    "postcss-html": "^0.23.6",
     "postcss-less": "^1.1.5",
+    "postcss-markdown": "^0.23.6",
```

Comment:
Likewise, wasn't `markdown` being removed for now, in this PR and to be added back in a PR of its own?

Reply:
Previously Markdown's processing logic was written in `postcss-html`, and now it is split into multiple packages: `postcss-syntax`, `postcss-html` and `postcss-markdown`